### ISSUE TITLE
Document the Projects workflow

### ISSUE THREAD

**Issue by wes-otf** [HyphaApp/hypha#4203]
It would be nice to have some in-depth documentation on the Projects workflow. To explain what configuration options exist, how each wagtail setting changes the workflow, etc would be really useful to cross reference as OTF deploys this along with new adopters coming along

**Comment by wes-otf** [comment]
I could see some sections for it under `User Guides` of https://docs.hypha.app

**Comment by frjo** [comment]
That would be very useful I think. @sandeepsajan0 has coded most of it.

Many of the features and functionalities was requested by people who no longer are at OTF.

**Comment by wes-otf** [comment]
yeah totally! would maybe help clean out some of the unneeded features too.
